### ISSUE TITLE
Strange output with seperated validation set

### ISSUE THREAD

**Issue by TimoK93** [fizyr/keras-retinanet#844]
Hey Team,

today i trained the first time with an seperated validation set. I used the parameter "--val-annotation" and an valid .csv file.

### Problem
There is a strange output after every epoch: Every precision and the mAP is 0.0000. The trained model is working quiet good.

The exact output is:

> Parsing annotations: 100% (1384 of 1384)
> 2057 instances of class one with average precision: 0.0000
> 13602 instances of class two with average precision: 0.0000
> mAP: 0.0000

### Version
I installed keras-retinanet master branch from 18.12.2018.

Greetings,
Timo

**Comment by hgaiser** [comment]
Hmm that's odd, I'll give it a shot on the latest master to see if I get the same issue. If you use your network to manually predict on images it gives you proper results? What happens if you run `evaluate.py` ?

**Comment by TimoK93** [comment]
Sorry for late response,

firstly I trained the model **without validation** set. Then I used that model as a pretrained model and retrained it with validation set on an **other machine**. There the bug occured...

Now I started a completely new training with the master from 18.12.2018 with validation set on the same machine. With this setup I cannot reproduce the behaviour...  I think it was an setup issue

**Comment by hgaiser** [comment]
Ah okay, thanks for coming back to let us know (and sorry, I forgot to test it).
